cmake/versions.cmake

```diff
@@ -1,9 +1,8 @@
 # Version for libraries CPP
-SET(VERSION "2.0.0")
+SET(VERSION "2.0")
```

Comment:
what should this be set to during development. The idea has been before that version 1.x.99 is a development version.

Reply:
That will still work. I just removed the reading of the third digit which was never used anywhere.

Reply:
ok